### DISCUSSION TITLE
摘要不会自动生成？

### DISCUSSION THREAD

**Discussion by cmpute** [CaiJimmy/hugo-theme-stack#759] [Q&A]
貌似主页显示的文章摘要只能通过`description`这个域来设置，文章主体的内容并不会自动显示在主页，不知道是否是我设置不对？

**Comment by CaiJimmy** [comment]
对的，就是这样。

**Reply by cmpute** [reply]
那我有点好奇，这行代码是相当于不会起作用吗？

https://github.com/CaiJimmy/hugo-theme-stack/blob/4272da7ee402b43fab957cf328e622279168354c/layouts/partials/data/description.html#L12-L15

**Reply by CaiJimmy** [reply]
抱歉这么久才回复。

`description.html` 只用来输出 `<meta name="description">`，主页显示的摘要只能通过 `description` 字段设置。

**Reply by cmpute** [reply]
感谢回复！知道了
